D/tests/syntax_test_d.d

```diff
@@ -1603,6 +1603,31 @@ extern(1)
 //          ^ punctuation.definition.string.end.d
 //           ^ punctuation.section.parens.end.d
 //            ^ punctuation.terminator.d
+  is(mixin("int") == int);
+//^^ keyword.other.d
+//  ^ punctuation.section.parens.begin.d
+//   ^^^^^ keyword.control.d
+//        ^ punctuation.section.parens.begin.d
+//         ^ meta.string.d string.quoted.double.d punctuation.definition.string.begin.d
+//          ^^^^ meta.string.d string.quoted.double.d
```

Comment:
Mostly a style thing, but over time I've found that using assertions such as the following tend to be easier to modify and understand:

```suggestion
//         ^^^^^ meta.string.d string.quoted.double.d
//         ^ punctuation.definition.string.begin.d
```